### ISSUE TITLE
Accessibility - Step 6 Topic 2 - Data Tables Need Correcting

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#57]
The four data TABLEs on the Step 6 Topic 2 should have the first row be the TABLEs CAPTION. Also, the second row which have the text "Circumstances" and "Remediation Actions" should be a TH (TABLE heading) with a scope="col" so VoiceOver and other screen reader users will know what a particular cell belongs too. As an example you want the screen reader user to be able to tell that the number 27 in the fourth column and twentieth row is to number of winners at the races.

Making these changes will help the website screen reader user out greatly along with the application too.

Similar changes should be done to any other TABLEs in the application.

**Comment by adriennerlong** [comment]
ICF, please make these changes and disregard the guidance ILAB provided in the marked-up word document.

**Comment by jfc3-dol** [comment]
Here is some slides of mine that have accessible TABLE examples, articles, etc,. in them that might be helpful.

http://jfciii.com/presentations/wasp/accessibility.html

To see next slide use Enter key or the arrow keys to go back and forth.

**Comment by jfc3-dol** [comment]
@adriennerlong.

The top line/cell that spanned two columns has been removed and assuming other accessibility to the table have been made. 

I did a quick check on website in Chrome and the TABLEs I checked had table headings of TH with a scope="col".

Closing issue.
